### PR TITLE
Dance Share Page: click to run image 

### PR DESCRIPTION
**Pull request by Erin007**
[Issue #377](https://github.com/code-dot-org/dance-party/issues/377)

Rather than showing a blank screen prior to the run button being clicked on the share page of a dance party project, we now have a snazzy image to remind the user to click run. 

click-to-run

Note:  Starwars and Flappy do this differently because they have backgrounds and skins.  As far as I can tell, Dance Party is different enough from those level types that consistency in implementation of a "click to run" themed watermark isn't crucial across project types.

### CODE REVIEW

File: apps/src/dance/Dance.js [hunk 1]
```diff
@@ -417,6 +418,7 @@ Dance.prototype.onReportComplete = function (response) {
  * Click the run button.  Start the program.
  */
 Dance.prototype.runButtonClick = async function () {
+  document.getElementById('danceClickToRun').style.display = "none";
```
Comment (islemaster) [inline apps/src/dance/Dance.js]: Does this break when not in share view?  I'd expect `document.getElementById('danceClickToRun')` to return `undefined` when we're not in share view.

Comment (Erin007) [reply apps/src/dance/Dance.js]: Yep, this broke when not in share view. Thanks for looking closely! I updated to only hide/show that element if it exists.

**Comment by Erin007** [comment]
@islemaster this is ready for another look when you've got a minute. Circle is much happier this time around and both the share view and non share view work - thanks for the feedback!

File: apps/src/dance/Dance.js (hunk 1 above)
Comment (ajosuarez) [reply apps/src/dance/Dance.js]: does this at all break or remove the need for the loading spinner?

**Comment by ajosuarez** [comment]
this is awesome, looks great btw! had one question about what this change does for the loading spinner - is there any impact there?

**Comment by Erin007** [comment]
dance-spinner

@simplycloud good question!  This doesn't interfere with the dancing spinner.  It still shows briefly (as shown in this gif - you may need to click on it to see, not sure why it's not autoplaying) while everything loads, then when the program is ready to run the new "click to run" image appears.

**Comment by islemaster** [comment]
:+1: I think the loading spinner is still important for the non-share case too.

**Pull request merged**